README.md

```diff
@@ -63,6 +63,45 @@ Requests to cross-scheme subresources will no longer send `SameSite=Lax` or `Sam
 For example, if http://site.example embeds an image from https://site.example, `SameSite=Lax` and `SameSite=Strict` cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
 ## Questions
+### How does this affect iframes?
+#### The src attribute
+A request to get the content specified by the `src` attribute is treated as any other subresource request and is affected the same ways.
+Once the content is loaded it runs as cross-site or same-site depending on if the `src` value was cross-scheme or not, respectively.
```

Comment:
It looks at the domain too. This also isn't quite right in funny cases. For example, if `src` is `about:blank` or it used `srcdoc`, the origin is inherited. And `sandbox` can force it into an opaque origin, I think. Going into all those details is probably too verbose here, but maybe you could talk about "the origin of the resulting document" and add in parenthesis "(typically this is the origin in the `src` attribute)".

SameSite also looks at the entire ancestor chain, so if the ancestor chain is already cross-site, it doesn't matter what happens here. Not sure how to work that in without getting too long.

Reply:
What do you mean by "It looks at the domain too"? Are you referring to "if the `src` value was cross-scheme..."? I'm using the definition of "cross-scheme" that I established on line 44.

Yeah I got stuck here a while trying to decide what I should/shouldn't include. I eventually decided to keep this scoped to "Schemeful Same-Site" and mostly avoid explaining other concepts. I'll throw in a couple reminders.

I your phrasing of "the origin of the resulting document", I'll work that in.